package.json

```diff
@@ -4,6 +4,9 @@
   "private": "true",
   "description": "Node schedule utilities for use with Renovate bot.",
   "homepage": "https://github.com/renovatebot/node-schedule",
+  "bugs": {
+    "url" : "https://github.com/renovatebot/node-schedule/issues"
+    },
```

Comment:
Maybe this is better, as it's better defined?
This is the format that Renovate bot uses on the main repo, so I assume that way works best?

https://github.com/renovatebot/renovate/blob/9247832066dbc0039dcc820765bb5bcfea370c4b/package.json#L112-L114